tests/utils/address_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from common.utils.address import format_addr, derive_vault_address
+from common.utils.receipt import PlanetID
```

Comment:
I cannot found `PlanetID` declaration in change list. Did I miss something?

Reply:
PlanetID added on #89